wd_demo.au3

```diff
@@ -141,35 +141,25 @@ Func _WD_Demo()
 						GUICtrlSetState($idButton_Run, @error ? $GUI_DISABLE : $GUI_ENABLE)
 					EndIf
 				Next
+
 		EndSwitch
 	WEnd
 
 	GUIDelete($hGUI)
 EndFunc   ;==>_WD_Demo
 
 Func RunDemo($idDebugging, $idBrowsers, $idUpdate, $idHeadless)
-	; Set debug level
+	; Check selected debuging demo opotion and set desired debug level
 	$_WD_DEBUG = $aDebugLevel[_GUICtrlComboBox_GetCurSel($idDebugging)][1]
 
-	#Region - WebeDriver update
-	Local $sUpdate
-	_GUICtrlComboBox_GetLBText($idUpdate, _GUICtrlComboBox_GetCurSel($idUpdate), $sUpdate)
-
-	Local $bFlag64 = (StringInStr($sUpdate, '64') > 0)
-	If StringInStr($sUpdate, 'Current') Then $bFlag64 = Default
-	Local $bForce = (StringInStr($sUpdate, 'Force') > 0)
-	If $sUpdate = 'Report only' Then $bForce = Null
+	; Get selected browser name demo option
+	Local $sBrowserName = $aBrowsers[_GUICtrlComboBox_GetCurSel($idBrowsers)][0]
 
-	Local $sBrowser = $aBrowsers[_GUICtrlComboBox_GetCurSel($idBrowsers)][0]
-	Local $bUpdateResult = _WD_UpdateDriver($sBrowser, @ScriptDir, $bFlag64, $bForce)
-	ConsoleWrite('$bUpdateResult = ' & $bUpdateResult & @CRLF)
-	#EndRegion - WebeDriver update
+	; Get selected update demo option and process update route
```

Comment:
This comment doesn't make sense to me, especially since you moved the code to a function. IMO, it would be better to say something like `;  Check & update WebDriver per user setting` or something like that.

Any further comments or clarification should be with the actual code performing the action.